packages/logging/src/api/payloadMiddleware.js

```diff
@@ -0,0 +1,10 @@
+export default function getStatePayloadMiddleware(getState) {
+	return function payloadMiddleware(payload) {
+		return {
+			time: new Date(),
```

Comment:
would change the name to clearly state that is a browser time and as such dependent on user localisation and os preferences.
Maybe use toJSON on the Date object instance to retrieve clean date format ? see https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Date/toJSON